api/handlers_spaces.go

```diff
@@ -119,13 +126,15 @@ func (s *server) SpaceGetHandler(w http.ResponseWriter, r *http.Request) {
 			handleError(w, errors.Wrap(err, msg))
 			return
 		}
+
 		// cache results
-		ResultsCache[account].Set(spaceID, out, cache.DefaultExpiration)
+		resultCache.SetDefault(spaceID, out)
```

Comment:
So is this the actual fix?

Reply:
No, this is just doing the same thing, but in the normalized way.  The actual fix was twofold.  In the deployment job we specify the values since if you don't, but it's in the template, deco (or `text/template` actually) inserts `<no value>`.  `<no value>` fails to parse as a `time.Duration` and results in the `cache.DefaultExpireTime` to be zero.  We also made the change to return an error (instead of logging a warning) when ParseDuration fails.

Reply:
time.Duration was oddly returning a negative number: -(2^63), or -9223372037

The negative number never Expired or Purged, similar to NoExpiration